features/step_definitions/web.rb

```diff
@@ -315,4 +315,18 @@
     })
     step %Q/the step should succeed/
   end 
+end
+
+Given /^admin enable demo dynamic plugin successfully$/ do
+  ensure_admin_tagged
+  full_version, major, minor = env.get_version(user: user)
+  plugin_image_tag = full_version.gsub('.','')
+  demo_plugin_image = 'quay.io/openshifttest/console-demo-plugin:' + plugin_image_tag
+  step %Q|I obtain test data file "templates/oc-manifest-demo-plugin.yaml"|
+  step %Q/I replace lines in "oc-manifest-demo-plugin.yaml":/, table(%{
+    | image: quay.io/vszocs/console-demo-plugin | image: #{demo_plugin_image} |
```

Comment:
Perhaps put `quay.io/openshifttest/console-demo-plugin` in the yaml file ?

Reply:
updated, PTAL